README.md

```diff
@@ -55,6 +55,13 @@ More information in [wiki page](https://github.com/adireddy/haxe-checkstyle/wiki
 			}
 		},
 		{
+			"type": "FileLength",
+			"props": {
+				"severity": "WARNING",
+				"maxLength": 1000
```

Comment:
Can you please change `maxLength` to `max` similar to http://checkstyle.sourceforge.net/config_sizes.html